### PR TITLE
fix: Icon add displayName

### PR DESCRIPTION
**Pull request by handycode**

### CODE REVIEW

File: components/icon/index.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export interface IconProps {
   style?: React.CSSProperties;
 }
 
-export default (props: IconProps) => {
+const icon = (props: IconProps) => {
```
Comment (yesmeck) [inline components/icon/index.tsx]: Uppercase first letter.

Comment (benjycui) [inline components/icon/index.tsx]: How about:

```jsx
function Icon() {
  ...
}
```

Comment (yesmeck) [inline components/icon/index.tsx]: @handycode

File: components/icon/index.tsx [hunk 2]
```diff
@@ -20,3 +20,7 @@ export default (props: IconProps) => {
   }, className);
   return <i {...omit(props, ['type', 'spin'])} className={classString} />;
 };
+
+(Icon as any).displayName = 'Icon';
```
Comment (afc163) [inline components/icon/index.tsx]: I think this line would be unnecessary

**Pull request merged**